docs/reference/analysis/analyzers.asciidoc

```diff
@@ -1,71 +1,148 @@
+search terms are represented in the same form as the indexed data.
 
-Elasticsearch comes with a number of prebuilt analyzers which are
-ready to use.  Alternatively, you can combine the built in
-character filters, tokenizers and token filters to create 
-<<analysis-custom-analyzer,custom analyzers>>.
+An analyzer contains one <<analysis-tokenizers,tokenizer>> and optional 
+<<analysis-charfilters,character>> and <<analysis-tokenfilters,token>> 
+filters. 
+
+image:images/AnalysisChain.png[]
+
+If present, _character filters_ preprocess the incoming text before
+passing it on to the tokenizer. For example, you can use character
+filters to strip out HTML tags or replace a particular character or 
+pattern of characters. 
+
+The _tokenizer_ splits the individual terms in the text apart on boundaries
+such as whitespace and punctuation and outputs a set of tokens that can be
+indexed. 
+
+If present, _token filters_ postprocess the set of tokens output by the 
+tokenizer. For example, you can use token filters to convert tokens to all lowercase,
+remove stopwords, perform stemming, inject synonyms, and generate bigrams for CJK characters.
+
+Elasticsearch provides several ready-to-use analyzers. 
+You can also construct <<analysis-custom-analyzer,custom analyzers>>
+from the built in character filters, tokenizers, and token filters. There are 
+also a number of https://github.com/elastic?query=elasticsearch-analysis[Elasticsearch analysis plugins] that add specialized text analysis options.
+
+You can specify which analyzer you want to use when configuring <<mapping, mappings>> and 
+<<search, searching>>. If you don't specify an analyzer, Elasticsearch uses the 
+appropriate <<default-analyzers,default analyzer>>.
+
+[float]
+[[configuring-analyzers]]
+=== Configuring Analyzers
+You can configure analyzers when you create an index and add analyzers to an 
+index with the <<indices-update-settings, `update settings`>> API. 
+
+TIP: While analyzers can also be configured at the node level, we recommend 
+configuring them at the index level to prevent the possibility of the configuration 
+getting out of sync across nodes. You need to tune your analysis options to suit the 
+content being analyzed, so analyzer configurations are naturally index-specific.
+
+To configure an analyzer when creating an index, you specify an analysis block
+in the index settings. For example, the following request creates the index
+`myindex` and configures the `standard` analyzer with the stopwords `a` and `an`.
+
+[source,js]
+--------------------------------------------------
+include::../snippets/analysis/configure-analyzer.json[]
+--------------------------------------------------
+// SENSE: analysis/configure-analyzer.json
+
+Similarly, you can add an analyzer to an existing index by updating the index
+settings. To do this, you need to close the index, add the analyzer, and then reopen the index.
+The following example adds a custom analyzer called `content` that
```

Comment:
I think we need a warning here... If you put the wrong settings into a closed index, it can put your index into an unrecoverable state, where you can't open the index with the existing settings, and you can't delete settings....  I'm wondering if we should really promote this way of applying analysis changes at all (or perhaps as an "experts only" section?)